### DISCUSSION TITLE
How to setup changeset for "thank you" messages

### DISCUSSION THREAD

**Discussion by konsalex** [changesets/changesets#1138] [Q&A] [answered]
Hey folks, how can I setup changeset so it can include the message `Thanks [@Andarist](https://github.com/Andarist)!...`, like in the release page: https://github.com/changesets/changesets/releases

<img width="1202" alt="CleanShot 2023-04-18 at 16 55 47@2x" src="https://user-images.githubusercontent.com/12672541/232799867-02cb4c6b-b053-42fa-aa07-279dd591b1a2.png">

In our case it creates a changelog with no link for commits and no special thank messages for the contributors
<img width="1238" alt="CleanShot 2023-04-18 at 16 55 58@2x" src="https://user-images.githubusercontent.com/12672541/232800094-3c6519c5-3852-4a5a-a7a5-fc345673670a.png">

**Comment by towry** [comment]
replace "Updated dependencies" with "Thanks @someone - Updated dependencies"

**Reply by konsalex** [reply]
@towry I think the functionality is native, but not sure how to use it:
https://github.com/changesets/changesets/blob/main/packages/changelog-github/src/index.ts#L115

any clue @Andarist can see you are the last contributor in the file 🤔

**Reply by towry** [reply]
@konsalex 

From the source code, it use regex to extract user from the summary: `/^\s*(?:author|user):\s*@?([^\s]+)/gim`, and I let chatgpt give me an example string that match this regex, it is `Author: @john_doe User: johndoe123`, you can try this.

**Reply by konsalex** [reply]
@towry my question is how I can set this up basically, not how it works.

**Reply by konsalex** [reply, marked as answer]
Just replying my question, as in this repo, a changelog config plugin is used: 

https://github.com/changesets/changesets/blob/main/.changeset/config.json#L2C3-L2C3
